### ISSUE TITLE
KCom2 - checkout and orders addresses

### ISSUE THREAD

**Issue by stephen-fueldesign** [Kademi/kademi-dev#6194]
I've done some testing and found that currently, the addresses appearing under Orders tab in Kademi does not seem to correlate between what was presented during checkout process and what the website visitor expects.

I'll describe my testing and present what data was captured and I would like feedback on any assumptions I've made, and what possibly needs to change on the front end and/or within Kademi itself so we can align all the processes.

**Here is the order I did**:
https://plantae.admin.kademi.com.au/shopping-carts/131371414/
- Shipping Address details are wrong.
- Billing Address is okay, as I checked "Enter different billing address"
- Billing Address: City field is not coming out in admin.
- Customer details are incorrect, as I was logged in as Stephen (_see below_)

**Here's the email Plantae got**:
https://plantae.admin.kademi.com.au/emails/131378575
- Shipping details are totally wrong
- name is wrong, or is not clear when typing in Jason Carr that this was the recipient's name
- phone is wrong, _same as above_

I tested as a logged in user, Steve.
Profile:
<img width="633" alt="1 profile" src="https://user-images.githubusercontent.com/39784677/47469818-62730d00-d85f-11e8-90b0-650118e7dd08.png">
**Profile Address** (used as shipping address):
<img width="626" alt="2 profileaddress" src="https://user-images.githubusercontent.com/39784677/47470013-520f6200-d860-11e8-8630-b019f7c308d6.png">

Checkout steps as follows:

**Shipping address**:
<img width="435" alt="1 shippingaddress" src="https://user-images.githubusercontent.com/39784677/47470445-445adc00-d862-11e8-9364-c18d4c516e14.png">

**Shipping destination**:
<img width="414" alt="2 shippingdestination" src="https://user-images.githubusercontent.com/39784677/47470452-4de44400-d862-11e8-89bd-728b7208c666.png">

**Credit Card number and address**:
<img width="420" alt="3 ccaddress" src="https://user-images.githubusercontent.com/39784677/47470463-5a689c80-d862-11e8-9a44-a8c14f85fd24.png">

**Credit Card number section - Billing address**:
<img width="416" alt="4 billingaddress" src="https://user-images.githubusercontent.com/39784677/47470475-68b6b880-d862-11e8-8dbf-72cd2b063a55.png">

**Comment by bradmac** [comment]
> Customer details are incorrect, as I was logged in as Stephen (see below)

The logged in user was correctly captured. The firstname/surname fields were being overwritten from payment details, which is fixed now.

![image](https://user-images.githubusercontent.com/826458/47474319-e4206600-d872-11e8-8126-b1e490c0ae62.png)

**Comment by bradmac** [comment]
> Here's the email Plantae got:

As above. Please check after applying the update.

**Comment by bradmac** [comment]
@Uafinder please test the checkout process with eway, ensuring that the eway payment details form fields do not overwrite customer information captured on the order.

**Comment by Uafinder** [comment]
well... if i correctly understand the case, i have a user Joe Bloggs
He ordered some stuff
On the payment details screen i entered different card holder, Jenny Rufus
Then when i go to order page:
No info about Jenny
![image](https://user-images.githubusercontent.com/1945230/48902533-88d99780-ee61-11e8-9240-4fc365ed5d23.png)
However when i go to Eway i'm seeing this
![image](https://user-images.githubusercontent.com/1945230/48902575-a149b200-ee61-11e8-95bf-0fd2e14c8d1a.png)

Is this a correct behavior?

**Comment by bradmac** [comment]
> On the payment details screen i entered different card holder, Jenny Rufus. Then when i go to order page: No info about Jenny

Yes, this is correct. The information entered for payment is completely independent of the order.

Ie the customer information on the order is not the Visa card holder information.

In the example above you should see information about Jenny Rufus only on the payment transactions page, not on the order

**Comment by bradmac** [comment]
@Uafinder note the requirement stated above..

> ensuring that the eway payment details form fields **do not** overwrite customer information captured on the order.
